Updated functionality for url_to_filename

MarijaAP opened the pull request:
Solving issue#21

nappex commented on freezeyt.py:
```diff
@@ -1,18 +1,30 @@
+    """
+    url_parse = urlparse(url)
+
+    if url.startswith('http'):
```
```suggestion
    if url.startswith('http') or url.startswith('https'):
```

nappex replied:
Maybe we could add 'https' protocol also.

encukou replied:
If the URL string `startswith('http')`, it also `startswith('https')`.
But, you can use `url_parse.scheme` instead of `startswith`.

nappex commented:
I deleted my comment with @encukou comment also by mistake. He wrote: *If the URL string startswith('http'), it also startswith('https'). But you can use url_pare.scheme instead of startswith.*

 I apologize for deletion. I did not notice that @encukou commented it.

MarijaAP commented:
> I deleted my comment with @encukou comment also by mistake. He wrote: _If the URL string startswith('http'), it also startswith('https'). But you can use url_pare.scheme instead of startswith._
> 
> I apologize for deletion. I did not notice that @encukou commented it.

Oki, so I am replying here to @encukou , I just changed .startswith to .scheme. Thanks
